apps/authentication/routes.py

```diff
@@ -241,6 +243,94 @@ def resend_code():
 
     return redirect(url_for('authentication_blueprint.confirm_page'))
 
+@blueprint.route('/reset-password', methods=['GET', 'POST'])
+def reset_password():
+    form = ResetPasswordForm(request.form)
+
+    if 'identifier' in request.form:
+        identifier = request.form['identifier']
+        user = Users.query.filter(or_(Users.email == identifier, Users.username == identifier)).first()
+        if not user:
+            return redirect(url_for('authentication_blueprint.confirm_reset_password'))
+  
+        # Send email
+        confirmation_token = secrets.token_hex(8)
+        session['confirmation_token'] = confirmation_token
+        session['user'] = user.email
+        session['datetime'] = utcnow()
+        session['try'] = 0	
+
+        msg = Message(
+            subject="HackInSDN - Reset your password",
+            sender=app.config['MAIL_USERNAME'],
+            recipients=[user.email],
+            body=(
+                "Help us protect your account\n\n"
+                "Before you reset your password, we need to verify your identity. Enter the following code on the sign-up page.\n\n"
+                f"{confirmation_token}\n\n"
+                "If you have not recently tried to sign up into HackInSDN, you can ignore this e-mail.\n\n"
+                "--\n\n"
+                f"You're receiving this email because of your account on Dashboard HackInSDN."
+            ),
+            html=render_template('mail/confirmation_token.html', confirmation_token=confirmation_token),
+        )
+        mail.send(msg)
+
+        return redirect(url_for('authentication_blueprint.confirm_reset_password'))
+
+    else:
+        return render_template('pages/reset_password.html', form=form )
+
+@blueprint.route('/confirm-reset-password', methods=['GET', 'POST'])
+def confirm_reset_password():
+    form = ResetPasswordConfirmForm(request.form)
+
+    confirmation_token = session.get('confirmation_token')
+
+    if not confirmation_token or not session.get('user') or not session.get('datetime') or session.get('try') == None:
+        db.session.add(LoginLogging(ipaddr=get_remote_addr(), login=session.get('user'), auth_provider="local", success=False, message="Invalid session parameters"))
+        db.session.commit()
```

Comment:
Here instead of adding to the DB, you should basically do this:

```suggestion
        app.logger.info(f"Invalid reset password request ipaddr={get_remote_addr()} login={session.get('user')} auth_provider=local")
```